.github/workflows/genesis-verify.yml

```diff
@@ -0,0 +1,57 @@
+name: genesis-verify
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - "misc/deployments/**/genesis.json"
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v41
+        with:
+          files: "misc/deployments/**/genesis.json"
+
+      - name: List changed genesis files
+        run: |
+          for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
+            echo "Changed file: $file"
+          done
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "1.22"
+
+      - name: Install dependencies
+        run: go mod download
+
+      - name: Build gnogenesis
+        run: make -C contribs/gnogenesis
+
+      - name: Verify each genesis file
+        run: |
+          for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
+            echo "Verifying $file"
+            gnogenesis verify -genesis-path $file
+          done
+
+      - name: Build gnoland
+        run: make -C gno.land install.gnoland
+
+      - name: Running latest gnoland with each genesis file
+        run: |
+          for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
+            echo "Running gnoland with $file"
+            gnoland start -lazy=true --genesis $file
+            # TODO: how to determine if the genesis is valid and chain did start?
```

Comment:
@sw360cab 
What do you think is the cleanest way to test this?

Maybe have the command not exit for like 60s?

Reply:
What if genesis.json has number of 10000000000 initial txs?? 60s might not be enough

Reply:
Can we bench it?

Reply:
I'm curious if we shove 1000000 txs in the genesis, what is the genesis generation time

Reply:
I think bench should be done by other pr. 
It really depends on..
1. what kind of tx we want to ( simple coin send vs deploying complex contract )
2. (perhaps) hardware spec that runs gnoland ( apple silicon vs raspberry pi )

Reply:
60s
9f3ec963edd74af104fe7e511d9355434763c3dd

Reply:
Did bit of bench with
- gnodev
- supernova.
initial txs of 5000, takes less than 10 seconds so I think 60s is enough